index.js

```diff
@@ -261,3 +305,25 @@ getParser(roomId, function (err, parseMessage) {
 		});
 	});
 });
+
+function retry (func) {
+	const maxTries = 3;
```

Comment:
in these cases, maxtries is generally a defaulted argument

retry (func, maxTries) {
  maxTries = maxTries || 3;